meetings/2023-01-18.md

```diff
@@ -0,0 +1,63 @@
+## TSC Meeting - 18th Jan 2023
+
+Meeting agenda: https://github.com/finos/technical-steering-committee/discussions/49
+
+## Attendees
+
+TSC members
+ - Colin Eberhardt, @ColinEberhardt, Scott Logic (chair)
+ - Cara Delia, @cdeliaRH, Red Hat
+ - Bruno Domingues, @brunodom, Intel
+ - Olivier Poupeney, @opoupeney, Symphony
+ - Stephen Goldbaum, @stephengoldbaumyes, MS
+ - Jon Freedman, @jonfreedman, Point 72
```

Comment:
I'm no longer with Point72, so either Brevan Howard or Member at Large depending on your preference please

Reply:
Ah yes, copy paste error. I’ll get that updated 👍